### PR TITLE
Feature - Updated NavigationBar to use ui-react

### PR DESCRIPTION
**Pull request by MoiyadKayda**
## Screenshots:
image

## Checklist:

- [ ] I have updated the CHANGELOG with a summary of my changes

### CODE REVIEW

File: src/packages/console/src/components/navigationBar.tsx [hunk 1]
```diff
@@ -2,19 +2,14 @@ import React from 'react';
 
 import { deepCompare } from '@kibalabs/core';
 import { useInitialization, useNavigator } from '@kibalabs/core-react';
-import { Box, Image } from '@kibalabs/ui-react';
+import { Alignment, Box, Button, Direction, Image, LinkBase, PaddingSize, Stack, Text } from '@kibalabs/ui-react';
 import AppBar from '@material-ui/core/AppBar';
```
Comment (krishan711) [inline src/packages/console/src/components/navigationBar.tsx]: we shoud get rid of all these material imports - whats preventing them from being removed?

Comment (MoiyadKayda) [reply src/packages/console/src/components/navigationBar.tsx]: what should I use to replace the AppBar and Toolbar

Comment (krishan711) [reply src/packages/console/src/components/navigationBar.tsx]: i dont know - what would you suggest?

File: src/packages/console/src/components/navigationBar.tsx [hunk 2]
```diff
@@ -70,56 +70,54 @@ export const NavigationBar = React.memo((): React.ReactElement => {
...
-            <Button
-              variant='outlined'
-              onClick={onResendVerificationClicked}
-            >
-              Resend Verification
-            </Button>
-          )}
-          {verificationSent && (
-            <Typography color='textPrimary'>
+        <BackgroundView color='#ffdd7e'>
```
Comment (krishan711) [inline src/packages/console/src/components/navigationBar.tsx]: please make this a color and use `$colors.banner` here instead of the raw value

**Review by krishan711 — APPROVED** [review]

**Pull request merged**